snippets/longest_word.rb

```diff
@@ -1,6 +1,6 @@
 def longest_word(sentence)
  longest = sentence.split
- longest.sort_by(&:length).reverse!
+ longest = longest.sort_by(&:length).reverse
```

Comment:
join line2 and 3 as one line.
something like:
```
longest = sentence.
            split.
            sort_by(&:length).
            reverse
```

Reply:
Good catch